### PR TITLE
fixing broken tox setup

### PR DESCRIPTION
**Pull request by cash**
Noticed the unit tests are all failing due to a missing dependency

### CODE REVIEW

**Comment by cash** [comment]
I see tox is also an issue...

File: dbbackup/tests/commands/test_dbrestore.py [hunk 1]
```diff
@@ -132,6 +133,7 @@ def setUp(self):
         HANDLED_FILES.clean()
         add_private_gpg()
 
+    @unittest.skip("Consistently fails")
```
Comment (ZuluPro) [inline dbbackup/tests/commands/test_dbrestore.py]: What is the issue here ?

**Comment by ZuluPro** [comment]
Hello @cash 
Thank you very for you contributions,

Basically I wanted dbbackup to helps people to upgrade from old Python/Django versions to newers.
But people using Py 3.2 and 3.3 appears creapy nowadays.

**Comment by mihow** [comment]
The tests are still failing for all PRs

**Comment by devkral** [comment]
I think the issues are mostly solved. Only the gpg and mongodb issue are still present.

**Comment by jonathan-s** [comment]
As this PR isn't relevant any longer I'm closing this. The tox setup is working at the moment.

**Pull request closed without merging**